lib/env_bang.rb

```diff
@@ -43,9 +46,11 @@ def values
 
     def [](var)
       var = var.to_s
-      raise KeyError.new("ENV_BANG is not configured to use var #{var}") unless vars.has_key?(var)
 
-      Classes.cast ENV[var], vars[var]
+      return Classes.cast ENV[var], vars[var] if vars.has_key?(var)
+      return if ENV['SKIP_ENV_ERRORS']
```

Comment:
I don't think we need this - otherwise we start eating errors when we should not (surely skipping an env and then requesting it via `ENV!` sounds like an issue?)